src/js/veteran-representative/2122-schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Appointment of Veterans Service Organization As Claimant's Representative",
+  "type": "object",
+  "definitions": {
+    "fullName": {
+      "type": "object",
+      "properties": {
+        "first": {
+          "type": "string"
+        },
+        "middle": {
+          "type": "string"
+        },
+        "last": {
+          "type": "string"
+        },
+        "suffix": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "properties": {
+    "veteranFullName": {
+      "type": "string"
+    },
+    "veteranSSN": {
+      "type": "string"
+    },
+    "vaFileNumber": {
+      "type": "string"
+    },
+    "insuranceNumber": {
+      "type": "string"
+    },
+    "claimantFullName": {
+      "type": "string"
+    },
+    "claimantAddress": {
+      "type": "string"
+    },
+    "claimantEmail": {
```

Comment:
This should also have a `format: 'email'` property, so that we take advantage of the default JSON Schema email validation logic.